test/with_dummyserver/test_connection.py

```diff
@@ -33,6 +34,32 @@ def test_returns_urllib3_HTTPResponse(pool: HTTPConnectionPool) -> None:
     assert isinstance(response, HTTPResponse)
 
 
+hook_reported = False
+
+
+def test_audit_event(pool: HTTPConnectionPool) -> None:
+    # Audit hooks are only available in versions >= 3.8
+    if not hasattr(sys, "audit"):
```

Comment:
We should be skipping this test if there's no `sys.audit`, you can use `pytest.mark.skipif` for this.